.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to pyccel
+
+Firstly, thanks for your interest in contributing code to pyccel!
+
+We welcome any and all contributions.
+
+There are many ways to help with the pyccel project which are more or less involved.
+The following is a summary of how you can do this.
+
+## Asking questions
+
+For simple questions (e.g. installation problems), we encourage you to use the [discussions tab](https://github.com/pyccel/pyccel/discussions/categories/q-a).
+
+For more involved questions we also have a [slack channel](pyccel.slack.com).
```

Comment:
This link to the Pyccel channel doesn't grant everyone access to the channel.
If you see that these kinds of questions need to be discussed in the Pyccel slack channel, then we need to figure out how to manage new users in Pyccel slack, we can share here an invitation link or safely we ask the user to send an email to the pyccel group in order to add him to slack.
@yguclu what do you think?

Reply:
@rakati I agree, in most cases the discussion tab should be sufficient.

In the future we may move from Slack in favor of Discord, where it should be easy to create public channels for the user community of Pyccel.

Reply:
Fixed 8d48253e